src/_data/sidenav.yml

```diff
@@ -126,6 +126,8 @@
           children:
             - title: Add a single Flutter screen
               permalink: /docs/development/platform-integration/add-to-app-android/add-flutter-screen
+            - title: Add a Splash Screen
```

Comment:
This is ok to merge for now. @Sfshaza do you have any thoughts on how big this subtree can get? @matthew-carroll at this rate I assume this subtree will have something like 5-10 articles? That seems ok to me. Otherwise, we can also generalize into a "Performance implications" type page and this could be a solution type to the general 'perceived latency' challenge, to which there are many solutions. 

Anyway, just food for thought. Nothing we need to do now.

Reply:
I think 5-10 articles is OK. But we’d probably not want it to grow a lot more than that.

Reply:
ok, let's leave the structure as is until we run into the size issue.